src/api/app/components/notification_component.html.haml

```diff
@@ -20,6 +20,19 @@
             = helpers.bs_request_state_badge(@notification.notifiable.state)
           - if @notification.notifiable_type == 'WorkflowRun'
             = render WorkflowRunStatusBadgeComponent.new(status: @notification.notifiable.status, css_class: 'ms-1')
+          - if @notification.notifiable_type == 'Report' && count_of_additional_reports_for_reportable >= 1
+            %span.badge.text-bg-info.ms-1
+              +#{count_of_additional_reports_for_reportable}
+              = 'Report'.pluralize(count_of_additional_reports_for_reportable)
+          - if @notification.notifiable_type == 'Report'
+            - if @notification.notifiable.decision.present?
+              %span.badge.text-bg-success.ms-1
+                %i.fas.fa-gavel.me-1
+                Decision made
```

Comment:
```suggestion
                Decided
```

Reply:
But which one? :)

Reply:
Which of the reports has a decision you mean? All the of the reports for the same reportable are associated to the same decision.

Reply:
Wording is addressed